resolve comments

wangmingliang-ms opened the pull request:
- rename getZOrder to getOrder
- normalize some class/method's names
- rename Action.Proxy -> Action.Delegate
- remove default implementation of IView#setUpdater & getUpdater.
- remove Action.emptyHandler() and EMPTY_HANDLER
- close properties view when app is deleted.
- add back missing telemetry for springcloud run configuration
- open test url action only applies to SpringCloudApp.
- fix custom messager in intellij

wangmingliang-ms committed:
rename getZOrder to getOrder

wangmingliang-ms committed:
normalize some class/method's names

wangmingliang-ms committed:
rename Action.Proxy -> Action.Delegate

wangmingliang-ms committed:
remove default implementation of IView#setUpdater & getUpdater.

wangmingliang-ms committed:
remove Action.emptyHandler() and EMPTY_HANDLER

wangmingliang-ms committed:
close properties view when app is deleted.

wangmingliang-ms committed:
add back missing telemetry for springcloud run configuration

wangmingliang-ms committed:
open test url action only applies to SpringCloudApp.

wangmingliang-ms committed:
fix custom messager in intellij

Flanker32 commented on PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/IntellijSpringCloudActionsContributor.java:
```diff
@@ -33,29 +33,29 @@ public void registerHandlers(AzureActionManager am) {
     @Override
-    public int getZOrder() {
-        return IActionsContributor.super.getZOrder();
+    public int getOrder() {
```
Do we need to add an override here?

Flanker32 commented on PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/deplolyment/SpringCloudDeploymentConfigurationState.java:
```diff
@@ -92,25 +112,35 @@ private void printPublicUrl(final SpringCloudApp app) {
-            out.accept(message);
-            super.error(message, title);
+        public boolean show(IAzureMessage raw) {
+            if (raw.getType() == IAzureMessage.Type.INFO) {
```
switch?

Flanker32 commented on PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/IntellijSpringCloudActionsContributor.java:
```diff
@@ -21,7 +21,7 @@
 import java.util.function.BiPredicate;
 
-public class IntellijSpringCloudActions implements IActionsContributor {
+public class IntellijSpringCloudActionsContributor implements IActionsContributor {
```
We may also need to add telemetry properties in action handler

wangmingliang-ms committed:
store only BiPredicate:BiConsumer

wangmingliang-ms committed:
rename action.title -> action.label
remove action.description and add action.title(:AzureString) which is same as IAzureOperation.title.

wangmingliang-ms committed:
perform action at background in default(adding error handling/telemetry feature)

wangmingliang-ms committed:
set subscriptionId and resourceType in telemetry for AzureResource related actions.

wangmingliang-ms committed:
remove unnecessary runOnPooledThread.

wangmingliang-ms committed:
speed up 'runIde'
